$OpenBSD: patch-gtkglext_gtkmm_gl_widget_cc,v 1.3 2016/09/07 20:29:52 ajacoutot Exp $

Fix build with gtk2mm>=2.24.5

avoid "error: 'GdkSpanFunc' has not been declared" by including
gtk/gtkglwidget.h before widget.h.

--- gtkglext/gtkmm/gl/widget.cc.orig	Wed Sep  7 22:19:40 2016
+++ gtkglext/gtkmm/gl/widget.cc	Wed Sep  7 22:19:51 2016
@@ -17,9 +17,10 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307  USA.
  */
 
-#include <gtk/gtkglwidget.h>
-
+#include <glibmm.h>
 #include "widget.h"
+
+#include <gtk/gtkglwidget.h>
 
 namespace Gtk
 {
